### PR TITLE
add section: SVG to raster format conversion

### PR DESCRIPTION
**Pull request by rishavbhowmik**
Add documentation for upcoming support of SVG to raster format conversion.

### CODE REVIEW

File: features/image-optimization/README.md [hunk 1]
```diff
@@ -11,3 +11,4 @@ The following features related to image optimization can be found in your dashbo
 3. [Chroma Subsampling](chroma-subsampling.md)
 4. [Metadata, Color Profile and Orientation](metadata-color-profile-and-orientation.md)
 5. [PNG Compression](png-compression.md)
+6. [SVG to raster format conversion](svg-to-raster-format-conversion.md)
```
Comment (violetviolinist) [inline features/image-optimization/README.md]: This should go under the `Image transformations` header imo. Converting SVGs to Raster formats isn't really an optimization.

File: features/image-optimization/svg-to-raster-format-conversion.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# SVG to raster format conversion
+
+ImageKit.io allows you to convert SVG images to raster formats by using the [format parameter (f)](../image-transformations/resize-crop-and-other-transformations.md#format---f).
```
Comment (violetviolinist) [inline features/image-optimization/svg-to-raster-format-conversion.md:3]: `format parameter` => `format transformation parameter`

File: features/image-optimization/svg-to-raster-format-conversion.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+To perform this conversion, you need to specify the desired raster format as `f-<raster_format>`. For example: `f-png` or `f-jpg` etc.
```
Comment (violetviolinist) [inline features/image-optimization/svg-to-raster-format-conversion.md]:
```suggestion
To perform this conversion, you need to specify the desired raster format as `f-<raster_format>` in the image URL. For example: `f-png`, `f-jpg`, etc.
```

File: features/image-optimization/svg-to-raster-format-conversion.md (hunk 2 above)
Comment (violetviolinist) [inline features/image-optimization/svg-to-raster-format-conversion.md]:
```suggestion
ImageKit.io allows you to convert SVG images to raster formats by using the [format transformation parameter (f)](../image-transformations/resize-crop-and-other-transformations.md#format---f).
```

File: features/image-optimization/svg-to-raster-format-conversion.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+{% hint style="info" %}
+If you enable automatic format conversion or use `f-auto`, the SVG images will not be converted to raster formats and will be delivered as SVG files only.
```
Comment (violetviolinist) [inline features/image-optimization/svg-to-raster-format-conversion.md]:
```suggestion
If you enable automatic format conversion or use `f-auto` in the image URL, SVG images will not be converted to raster formats and will be delivered as SVG files only. Specifying `f-<raster_format>` in the URL is the only way to convert SVGs to raster formats.
```

Comment (violetviolinist) [reply features/image-optimization/svg-to-raster-format-conversion.md]: Also, add a link to the `Automatic format conversion` page

File: features/image-optimization/svg-to-raster-format-conversion.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+{% endhint %}
+
+Once the SVG images are converted to raster formats, you can also use other image transformation parameters to resize, filter, and add overlays.
+
+{% hint style="info" %}
+SVG to SVG conversion does not support image transformations (such as resizing, adding overlay and applying filters) on SVG images directly.
```
Comment (violetviolinist) [inline features/image-optimization/svg-to-raster-format-conversion.md]:
```suggestion
To apply image transformations such as resizing, adding overlays, etc., SVG images must be converted to raster formats.
```

File: features/image-optimization/svg-to-raster-format-conversion.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+ImageKit.io allows you to convert SVG images to raster formats by using the [format parameter (f)](../image-transformations/resize-crop-and-other-transformations.md#format---f).
+
+To perform this conversion, you need to specify the desired raster format as `f-<raster_format>`. For example: `f-png` or `f-jpg` etc.
+
+{% hint style="info" %}
+If you enable automatic format conversion or use `f-auto`, the SVG images will not be converted to raster formats and will be delivered as SVG files only.
+{% endhint %}
+
+Once the SVG images are converted to raster formats, you can also use other image transformation parameters to resize, filter, and add overlays.
```
Comment (violetviolinist) [inline features/image-optimization/svg-to-raster-format-conversion.md]:
```suggestion
When converting an SVG image to any raster format, you can also apply other image transformations such as resizing, overlays, etc.
```

File: features/image-optimization/README.md (hunk 1 above)
Comment (rishavbhowmik) [reply features/image-optimization/README.md]: Moved the section to README of image-transformations

Please check this screenshoot for overall diff in the text

Screenshot 2023-12-29 at 10 59 00 AM

File: features/image-optimization/svg-to-raster-format-conversion.md (hunk 2 above)
Comment (rishavbhowmik) [reply features/image-optimization/svg-to-raster-format-conversion.md]: https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069

File: features/image-optimization/svg-to-raster-format-conversion.md (hunk 4 above)
Comment (rishavbhowmik) [reply features/image-optimization/svg-to-raster-format-conversion.md]: https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069

File: features/image-optimization/svg-to-raster-format-conversion.md (hunk 5 above)
Comment (rishavbhowmik) [reply features/image-optimization/svg-to-raster-format-conversion.md]: https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069

File: features/image-optimization/svg-to-raster-format-conversion.md (hunk 6 above)
Comment (rishavbhowmik) [reply features/image-optimization/svg-to-raster-format-conversion.md]: https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069

File: features/image-optimization/svg-to-raster-format-conversion.md (hunk 3 above)
Comment (rishavbhowmik) [reply features/image-optimization/svg-to-raster-format-conversion.md]: https://github.com/imagekitio/docs-keep-a-changelog/pull/237#discussion_r1437996069

File: features/image-optimization/README.md (hunk 1 above)
Comment (violetviolinist) [reply features/image-optimization/README.md]: I meant a different page for this, but under the `Image transformations` parent heading

File: features/image-transformations/README.md [hunk 7]
```diff
@@ -97,3 +97,19 @@ You can overlay text on an image and create dynamic banners. You can also contro
...
+
+ImageKit.io allows you to convert SVG images to raster formats by using the [format transformation parameter (f)](../image-transformations/resize-crop-and-other-transformations.md#format---f).
+
+To perform this conversion, you need to specify the desired raster format as `f-<raster_format>` in the image URL. For example: `f-png`, `f-jpg`, etc.
+
+{% hint style="info" %}
+If you enable automatic format conversion or use `f-auto`, the SVG images will not be converted to raster formats and will be delivered as SVG files only.
+{% endhint %}
+
+If you enable [automatic format conversion](../image-optimization/automatic-image-format-conversion.md) or use `f-auto` in the image URL, SVG images will not be converted to raster formats and will be delivered as SVG files only. Specifying `f-<raster_format>` in the URL is the only way to convert SVGs to raster formats.
```
Comment (violetviolinist) [inline features/image-transformations/README.md:107]: This line is the same as the `info` box right above.

Comment (rishavbhowmik) [reply features/image-transformations/README.md:107]: removed the infobox

**Pull request merged**